### PR TITLE
Add patch application docs

### PR DESCRIPTION
**Pull request by krulwich**
https://greenhouseio.atlassian.net/browse/GREEN-6196

### CODE REVIEW

File: source/includes/harvest/_applications.md [hunk 1]
```diff
@@ -424,3 +424,95 @@ id | ID of the application to unreject
...
+Header | Description
+--------- | -----------
+On-Behalf-Of | ID of the user issuing this request. Required for auditing purposes.
+
+
+### JSON Body Parameters
+
+Parameter | Required | Type | Description
+--------- | ----------- | ----------- | ----------- | -----------
+related_post_id | No | integer | The ID of the reason why this application was rejected.
```
Comment (robnormal) [inline source/includes/harvest/_applications.md]: Uh...

Comment (krulwich) [inline source/includes/harvest/_applications.md]: woops!

File: source/includes/harvest/_applications.md [hunk 2]
```diff
@@ -424,3 +424,95 @@ id | ID of the application to unreject
...
+related_post_type | No | integer | The type of the application's related post
+source_id | No | integer | The ID of the application's source
+stage_status_id | No | integer | The ID of the application's stage status
+candidate_rejection_reason_id | No | integer | The ID of the application's candidate's rejection reason
+current_application_stage_id | No | integer | The ID of the application's current stage
+rejection_note_id | No | integer | The ID of the application's rejection note
+rejected_at | No | datetime | The date and time at which the application was rejected
+needs_decision | No | boolean | Boolean indicating whether the application needs a hiring decision
+referrer_id | No | integer | The ID of candidate's referer, if applicable
+prospect | No | boolean | A boolean indicating whether this application represents a candidate or a prospect
```
Comment (robnormal) [inline source/includes/harvest/_applications.md]: Would it be clearer to say "A boolean indicating whether this application represents a prospect (true) or a candidate (false)"

Comment (krulwich) [inline source/includes/harvest/_applications.md]: makes sense

File: source/includes/harvest/_applications.md [hunk 3]
```diff
@@ -424,3 +424,95 @@ id | ID of the application to unreject
...
+idle_after | No | datetime | How much longer than the application's interview_stage's 'dwell time' the application has been in that stage
```
Comment (robnormal) [inline source/includes/harvest/_applications.md]: I don't think this is a correct explanation of `idle_after`. It's a date and time, not a time period.

File: source/includes/harvest/_applications.md (hunk 2 above)
Comment (robnormal) [inline source/includes/harvest/_applications.md]: Or, "A boolean indicating whether this application represents a prospect instead of a candidate"

**Comment by robnormal** [comment]
Ship it. :egg:

**Pull request merged**